src/main/java/org/terasology/fluid/system/FluidClientSystem.java

```diff
@@ -53,8 +61,8 @@
     public void setItemTooltip(GetItemTooltip event, EntityRef container, FluidContainerItemComponent fluidContainerItem) {
         // Add tooltip with current fluid amounts.
         if (fluidContainerItem.fluidType != null) {
-            event.getTooltipLines().add(new TooltipLine("This holds " + (int) (fluidContainerItem.volume * 1000) + "/" +
```

Comment:
This extra 1000 was probably used to convert litres into ml, but I think as of now, the fluidContainerItems use ml..
https://github.com/Terasology/ManualLabor/blob/6314132942b9503c95b6198552b3713e75eee389/assets/prefabs/items/WoodenCup.prefab#L32